server/server.js

```diff
@@ -13,10 +15,41 @@ app.get('/products/:shoeId/gallery', (req, res) => {
       res.send(data);
     })
     .catch((err) => {
-      console.log(err);
       res.sendStatus(404);
     });
 });
+//create
+app.post('/products/:shoeId/gallery', jsonParser,(req, res) => {
+  let { shoeId } = req.params;
+  controller.post.productImages(shoeId, req.body.imageUrl)
+      .then((data) => {
+        res.sendStatus(201)
+      })
+    .catch((err) => {
+      res.sendStatus(404);
+    });
+});
+
+//update
+app.put('/products/:shoeId/gallery/:imageId', jsonParser, (req,res)=>{
+  let {shoeId, imageId} = req.params;
+  controller.put.productImages(shoeId, imageId, req.body.imageUrl)
+    .then((data)=> {
+      res.sendStatus(404)
```

Comment:
What status code do you intend to send in the case where the update was successful?

Reply:
Hi  Michelle, 

I had updated to a res.sendStatus(204) for a successful. Didn't seem to update. Will be included in the pull request I make tonight.